code/game/objects/structures/inflatable.dm

```diff
@@ -111,6 +111,13 @@
 	if(health <= 0)
 		deflate(1)
 
+/obj/structure/inflatable/AltClick()
+	if(usr.stat || usr.restrained())
+		return
+	if (cheap_hypotenuse(x, y, usr.x, usr.y) >= 2) //allow only 1 dist and diagonals
```

Comment:
This isn't the proper way to handle this; you'd want to use the the Adjacent proc.